code/__DEFINES/controllers/dbcore.dm

```diff
@@ -17,6 +17,11 @@
  */
 #define DB_MINOR_VERSION 3
 
+//* Tables *//
+
+/// Prefixes are currently disabled.
```

Comment:
tl;dr

1. citrp will not be doing cluster setups in the foreseeable future
2. even if it did, databases should be agnostic to which server is accessing it and be otherwise synchronous as we're kind of one-world
3. therefore there's no point in prefixing. it's bad practice to be sharing a database with other services anyways.